### DISCUSSION TITLE
Full Uninstall

### DISCUSSION THREAD

**Discussion by 000Lex000** [Sir-Walrus/Flips#59] [General]
How to remove COMPLETELY this utility from Windows OS?
To be more specific: 
All files with bps. extension get flower icon on them and get labeled as Flips utility file in their properties.
How to restore the default state of these files?
Thank you in advance.

**Comment by Sir-Walrus** [comment]
Those icons are controlled by the folders `.ips` `.bps` `FloatingIPSFileIPS` and `FloatingIPSFileBPS` in the registry folder `HKEY_CURRENT_USER/Software/Classes/`. You can delete them with regedit, but I don't know how quickly that will remove the Floating IPS File name; I think a reboot will complete the deletion, but I'm not entirely sure, I've never tried.

If you want to delete all traces of Flips, you'll also want to delete the hidden file floatingcfg.bin beside the Flips executable. Either show hidden files, or simply delete the entire folder.

There's no automatic way to do that, sorry.

I vaguely remember Windows XP offering to delete file associations if you simply delete the applicable .exe and double click the file, but I don't know if that still works on newer Windows, and I don't know if it removes the Floating IPS File name.

**Reply by 000Lex000** [reply]
Thank you very much for answering. That resolved the issue indeed.
IMHO you should add some sort of uninstaller or at least mention these things about regedit in README file, because this app is to intrusive for a small utility.
